src/Nethereum.RPC/Eth/EthGetCode.cs

```diff
@@ -49,12 +49,17 @@ public EthGetCode(IClient client) : base(client, ApiMethods.eth_getCode.ToString
             return base.SendRequestAsync(id, cancellationToken, address.EnsureHexPrefix(), block);
         }
 
-        public Task<string> SendRequestAsync(string address,
+        public async Task<string> SendRequestAsync(string address,
```

Comment:
@webwarrior-ws is this change related to this PR?

Reply:
yes

Reply:
if it was working and compiling before without the async, why do we need it now for this change?

Reply:
Because we need to await for result before we can check its type and act accordingly. The type of method doesn't change.

Reply:
ah fair enough, I overlooked that